ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -169,15 +224,34 @@ export const AccessRenewalPage = fp.flow(
   const [publications, setPublications] = useState<boolean>(null);
   const noReportId = useId();
   const reportId = useId();
+  const [reloadDisabled, setReloadDisabled] = useState(true);
```

Comment:
how does this work?  I don't see logic to disable reload here.  Should this be `disableDUCC` or something like that?

Reply:
Once enabled, I do not disable the button. If the status update succeeds (the moodle update returns that they are compliant) the button will disappear.

Reply:
OK I see now.  I was confusing reload Profile with reload Compliance ... maybe this needs a rename to distinguish these cases.